targets/REV3-LVSS/main.cpp

```diff
@@ -91,9 +93,11 @@ int main() {
         log::LOGGER.log(log::Logger::LogLevel::INFO, "Connected to CAN network\r\n");
     }
 
-    // Initialize all the CANOpen drivers.
+    // Initialize all the CANOpen dev.
     IO::initializeCANopenDriver(&canOpenQueue, &can, &timer, &canStackDriver, &nvmDriver, &timerDriver, &canDriver);
 
+    LVSS::LVSS lvss = LVSS::LVSS(reinterpret_cast<LVSS::TPS2HB50BQ1**>(powerSwitches));
```

Comment:
This isn't going to work. You never actually constructed the power switch instances. This `reinterpret_cast` is going to cause errors at runtime.